WeWatch/WeWatch/ContentView.swift

```diff
@@ -10,12 +10,11 @@ import SwiftUI
 struct ContentView: View {
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
+            
             Text("Hello, world!")
+                .font(Font.poppinsBold30px)
```

Comment:
You can omit using full version: `Font. poppinsBold30px` and use `.poppinsBold30px` instead.